### PR TITLE
Replace "Options" with user's email address (redux)

### PR DESCRIPTION
**Pull request by xen0m29**
Task Undertaken

- #70 Replace "Options" with user's email address (redux)

### CODE REVIEW

File: front/src/components/header/Header.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ import logo from 'images/logo.svg';
 
 interface StateProps {
   isLoggedIn: boolean;
+  email: string;
```
Comment (trybick) [inline front/src/components/header/Header.tsx:29]: nit - put email above isLoggedIn (alphabetize)

File: front/src/components/header/Header.tsx [hunk 2]
```diff
@@ -187,6 +188,7 @@ const mapStateToProps: MapStateToProps<StateProps, {}, AppState> = (
   state: AppState
 ): StateProps => ({
   isLoggedIn: selectIsLoggedIn(state),
+  email: state.user.email,
```
Comment (trybick) [inline front/src/components/header/Header.tsx]: Do you mind creating a selector, similar to `selectIsLoggedIn` here?

**Review by trybick — COMMENTED** [review]
Great job @xen0m29 ! I have two comments

File: front/src/components/header/Header.tsx (hunk 2 above)
Comment (xen0m29) [reply front/src/components/header/Header.tsx]: no problem, so what kind of selector do you want?

Comment (trybick) [reply front/src/components/header/Header.tsx]: It can look just like `selectIsLoggedIn` in `user/selectors.ts`

Comment (xen0m29) [reply front/src/components/header/Header.tsx]: ok, is there a preference for the name?

Comment (trybick) [reply front/src/components/header/Header.tsx]: Maybe `selectUserEmail`

**Comment by trybick** [comment]
Awesome @xen0m29 Thank you!

**Pull request merged**